neofs/config.yml

```diff
@@ -1,16 +1,6 @@
 name: "NeoFS"
 safemethods: ["alphabetList", "alphabetAddress", "innerRingCandidates", "config", "listConfig", "version"]
 events:
-  - name: Deposit
```

Comment:
This event produced by `OnNEP17Payment` function to track deposits, thus it is still there.